src/js/controllers/index.js

```diff
@@ -549,9 +548,8 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     c.startScan({
       includeCopayerBranches: true,
     }, function(err) {
-      if (err) {
-        if (self.walletId == walletId)
-          self.setOngoingProcess('scanning', false);
+      if (err && self.walletId == walletId) {
```

Comment:
That is exactly what does. If wallet is focused, display an error. If don't, not. For example, if we are trying to scan a wallet (not focused) that doesn't exist on the new BWS, it displays the message to recreate in the *focused* wallet.